src/main/java/mate/academy/onlinebookstore/entity/Category.java

```diff
@@ -0,0 +1,27 @@
+package mate.academy.onlinebookstore.entity;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+import lombok.Data;
+import org.hibernate.annotations.SQLDelete;
+import org.hibernate.annotations.Where;
+
+@Entity
+@Data
+@SQLDelete(sql = "UPDATE category SET is_deleted = true WHERE id=?")
+@Where(clause = "is_deleted=false")
```

Comment:
```suggestion
@SQLDelete(sql = "UPDATE category SET is_deleted = TRUE WHERE id = ?")
@Where(clause = "is_deleted = FALSE")
```
true and false are SQL keywords too